universal-application-tool-0.0.1/app/models/Applicant.java

```diff
@@ -0,0 +1,22 @@
+package models;
+
+import io.ebean.annotation.DbJsonB;
+import java.util.Map;
+import javax.persistence.Entity;
+import play.data.validation.Constraints;
+
+@Entity
+/** The ebeans mapped class that represents an individual applicant */
+public class Applicant extends BaseModel {
+  private static final long serialVersionUID = 1;
+
+  @Constraints.Required @DbJsonB
+  // When we build an object that Jackson can deserialize, we replace Map<String, Object> with that
+  // type.
+  // For now, this will be automatically deserialized - with subobjects being "Map<String, Object>"
+  // and lists
+  // being List<Object>.
+  public Map<String, Object> object;
```

Comment:
Thought (not-blocking): we'll probably want to access this directly as a string so we can parse it into a jsonpath `DocumentContext`. One way to do that is probably to implement a separate public method that provides access to the jsonpath-parsed `object`. I wonder if we can hook the ebean `save` method or override something so that we can tell the entity to serialize from the custom method.

Reply:
Sure, I've made a -> string method.  There's no obvious way to get the json string directly in the model object without deserializing and reserializing, but this should be all right for now.  When we build Jackson objects, we'll replace the implementation of these functions with Jackson serialization.